code/game/machinery/telecomms/broadcaster.dm

```diff
@@ -244,7 +244,20 @@ var/message_delay = 0 // To make sure restarting the recentmessages list is kept
 	else if(data == 2)
 
 		for(var/obj/item/device/radio/R in all_radios["[freq]"])
-			if(istype(R, /obj/item/device/radio/headset))
+			if(R.subspace_transmission)
+				continue
+
+			if(R.receive_range(freq, level) > -1)
+				radios += R
+
+	// --- This space left blank for Syndicate data ---
+
+	// --- Centcom radio, yo. ---
+
+	else if(data == 4)

```

Comment:
This, right here, along with the other one, is what broke NTSL. Data set to four is for *Artificial Transmissions*, I.E. signals sent by scripts. In addition, this bypasses normal tcomms signal logging due to this. I will need to change it in order to fix NTSL.